### ISSUE TITLE
Implementation for custom GA4 analytic tracking

### ISSUE THREAD

**Issue by ZoeBeykirch** [Netcentric/vg-volvotrucks-us-rd#172]
As a business owner and GA4 user, I want to be able to track how users are interacting with my digital reveal and other campaign pages. When a site visitor clicks on a media campaign from LinkedIn/Meta/Variant, and they click on the "Notify Me" and they complete the digital reveal form, I want to be able to know which media source the user came from. 

Acceptance Criteria: 
- Given that I click on a LinkedIn media asset that links to the digital reveal page, when I land on the reveal page, and click notify me, and fill out the digital reveal form, then I should be able to see that the user who completed the form came from the LinkedIn media asset. To add the LinkedIn Insights Tag to GTM more details on how to add the tag are in this link: https://www.linkedin.com/help/lms/answer/a416960
- GA4 custom tagging should first be implemented on the Digital Reveal page, and then the new VNL PDP (which can be done in a separate sprint). This custom tagging implementation should serve as a template framework for future campaign page needs. 

Analytics Airtable: [Click Here](https://airtable.com/appEUqWzJYOOAY8Hi/shrtMWGkLOZqdsd7V) 
Figma Analytics Maps: [Click Here](https://www.figma.com/file/kdaxj7iykUodGXERPozU9s/Analytics?type=design&mode=design&t=37bWqzWkE4kgnuN9-1)

**Comment by BeckyMedlin** [comment]
@ZoeBeykirch Please update requirements when ready.

**Comment by DanielaPedrochevd** [comment]
@ZoeBeykirch Can I and @manuel-vara get access to:

Analytics Airtable: [Click Here](https://airtable.com/appEUqWzJYOOAY8Hi/shrtMWGkLOZqdsd7V)
Figma Analytics Maps: [Click Here](https://www.figma.com/file/kdaxj7iykUodGXERPozU9s/Analytics?type=design&mode=design&t=37bWqzWkE4kgnuN9-1)

? 

Thanks!

**Comment by ZoeBeykirch** [comment]
Yes I have requested @ozzy-krishnan to share access with you both via email [daniela.pedroche@netcentric.biz](mailto:daniela.pedroche@netcentric.biz) and [manuel.vara@netcentric.biz](mailto:manuel.vara@netcentric.biz)

**Comment by DanielaPedrochevd** [comment]
@manuel-vara please align with @ozzy-krishnan  and @ZoeBeykirch whether they have tested the tag that has been already implemented and if it's working correctly.
